features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml.j2

```diff
@@ -59,10 +59,886 @@
+       <Scopes>
+            <Feature>
+                <Scope name="console:org:branding"/>
+            </Feature>
+            <Update>
+                <Scope name="internal_org_branding_preference_update"/>
+            </Update>
+            <Delete>
+                <Scope name="internal_org_branding_preference_update"/>
+            </Delete>
+            <Create>
+                <Scope name="internal_org_branding_preference_update"/>
+            </Create>
+            <Read>
+                <Scope name="internal_org_application_mgt_view"/>
+            </Read>
+        </Scopes>
+    </APIResourceCollection>
+    <APIResourceCollection name="org_emailTemplates" displayName="Email Templates" type="organization" version="v0">
+       <Scopes>
+            <Feature>
+                <Scope name="console:org:emailTemplates"/>
+            </Feature>
+            <Update>
+            </Update>
+            <Delete>
+            </Delete>
+            <Create>
+            </Create>
+            <Read>
+                <Scope name="internal_org_email_mgt_view"/>
+            </Read>
+        </Scopes>
+    </APIResourceCollection>
+    <APIResourceCollection name="org_smsTemplates" displayName="SMS Templates" type="organization" version="v0">
+       <Scopes>
+            <Feature>
+                <Scope name="console:org:smsTemplates"/>
+            </Feature>
+            <Update>
+            </Update>
+            <Delete>
+            </Delete>
+            <Create>
+            </Create>
+            <Read>
+                <Scope name="internal_org_template_mgt_view"/>
+            </Read>
+        </Scopes>
+    </APIResourceCollection>
+    <APIResourceCollection name="org_gettingStarted" displayName="Home Page" type="organization" version="v0">
+       <Scopes>
+            <Feature>
+                <Scope name="console:org:home"/>
+            </Feature>
+            <Update>
+                <Scope name="internal_org_application_mgt_update"/>
+            </Update>
+            <Delete/>
+            <Create>
+                <Scope name="internal_org_application_mgt_create"/>
+                <Scope name="internal_org_idp_create"/>
+                <Scope name="internal_org_user_mgt_create"/>
+                <Scope name="internal_org_group_mgt_create"/>
+            </Create>
+            <Read>
+                <Scope name="internal_org_application_mgt_view"/>
+            </Read>
+        </Scopes>
+    </APIResourceCollection>
+    <APIResourceCollection name="org_apiResources" displayName="API Resources" type="organization" version="v0">
+       <Scopes>
+            <Feature>
+                <Scope name="console:org:apiResources"/>
+            </Feature>
+            <Read>
+                <Scope name="internal_org_api_resource_view"/>
+            </Read>
+        </Scopes>
+    </APIResourceCollection>
+    <APIResourceCollection name="org_userStores" displayName="User Stores" type="organization" version="v0">
```

Comment:
this collection and the below `org_attributeDialects` collection was newly introduced